gamemodes/nzombies3/gamemode/effects/sv_tesla.lua

```diff
@@ -41,7 +41,7 @@
 --returns: boolean
 --
 
-function nz.Effects.Functions.Tesla( data )
+function Effects:Tesla( data )
```

Comment:
Well we should really do more commenting tbh. the tesla thing is probably the only thing I documented correctly.